content/en/community/_index.md

```diff
@@ -16,6 +16,6 @@ our [community repo](https://github.com/open-telemetry/community).
 
 ## Open Source Collaboration
 
-OpenTelemetry collaborates and integrates with other open-source projects
+OpenTelemetry collaborates and integrates with other open source projects
```

Comment:
In this case I'd say that the original is correct.

Reply:
I applied this change because of https://github.com/cncf/foundation/blob/main/style-guide.md

Reply:
Ok, that works for me.